src/app/hooks/useRbfTransactionData.ts

```diff
@@ -0,0 +1,274 @@
+import {
+  BtcTransactionData,
+  mempoolApi,
+  rbf,
+  RecommendedFeeResponse,
+  SettingsNetwork,
+  StacksTransaction,
+  stxToMicrostacks,
+  StxTransactionData,
+} from '@secretkeylabs/xverse-core';
+import { deserializeTransaction, estimateTransaction } from '@stacks/transactions';
+import { isLedgerAccount, microStxToStx } from '@utils/helper';
```

Comment:
nit: xverse-core already exports stxToMicrostacks and microstacksToStx
```suggestion
  stxToMicrostacks,
  microstacksToStx
  StxTransactionData,
} from '@secretkeylabs/xverse-core';
import { deserializeTransaction, estimateTransaction } from '@stacks/transactions';
import { isLedgerAccount } from '@utils/helper';
```